magprime/__init__.py

```diff
@@ -42,6 +42,11 @@ class Attendee:
     def donation_cost(self):
         return self.extra_donation if self.extra_donation else 0
 
```

Comment:
Two things:

1) Saying ``default=0`` in the ``Column`` definition above should already take care of this.  Like if you were to say ``Attendee().extra_donation`` then it should return ``0``.  The only situation this would cover is if the value was changed to some falsey value other than the default, like if you set it to ``None`` and then tried to save it,

2) Counter-intuitively, the ``@Session.model_mixin`` decorator is NOT applying this function anyway, because its name starts with an underscore.  I should probably remove this restriction and instead have it allow function names starting with one underscore (which should generally be okay to mix in) but disallow function names starting with two underscores (which would generally be dangerous to mix in).

So I'm not sure this PR actually does anything.